apps/extension/src/core/handlers/Extension.ts

```diff
@@ -97,12 +101,43 @@ export default class Extension extends ExtensionHandler {
       DEBUG || TEST ? 0 : 2000
     )
 
+    // setup polling for config from github
+    let CONFIG_UPDATE_INTERVAL = 1000 * 60 * 5 // 5 minutes
+    const MAX_CONFIG_UPDATE_INTERVAL = 1000 * 60 * 60 // 1 hour
+    setInterval(() => {
+      try {
+        this.getRemoteConfig()
+      } catch (e) {
+        // exponential backoff for rate limits
+        if (
+          (e as Error).message === CONFIG_RATE_LIMIT_ERROR &&
+          CONFIG_UPDATE_INTERVAL < MAX_CONFIG_UPDATE_INTERVAL
+        )
+          CONFIG_UPDATE_INTERVAL = CONFIG_UPDATE_INTERVAL / 2
+        else Sentry.captureException(e)
+      }
+    }, CONFIG_UPDATE_INTERVAL)
+
+    setTimeout(async () => {
+      this.getRemoteConfig()
+    }, 1000) // initial call immediately after extension start
+
     this.initDb()
     this.initWalletFunding()
     this.checkSpiritKeyOwnership()
     this.cleanup()
   }
 
+  private async getRemoteConfig() {
+    const config = await getConfig()
+
+    featuresStore.set({
```

Comment:
```suggestion
    await featuresStore.set({
```

Reply:
if method doesn't return anything, might want to rename it to `updateRemoteConfig`

Reply:
renamed to `fetchRemoteConfig`